pageserver/src/tenant.rs

```diff
@@ -1511,8 +1515,8 @@ impl Tenant {
                         timeline.set_state(TimelineState::Stopping);
                     }
                 }
-                TenantState::Broken => {
-                    info!("Cannot set tenant to Stopping state, it is already in Broken state");
+                TenantState::Broken{..} => {
+                    info!("Cannot set tenant to Stopping state, it is already in Broken state due to: {:?}", current_state);
```

Comment:
Could use `{current_state:?}` but this will print out:

```
Cannot set tenant to Stopping state, it is already in Broken state due to: Broken { custom debug impl with full backtrace }
```

Is that wanted (the full backtrace)?

I guess I am also a bit unclear why do we need a backtrace on broken state.

Also note the formatting on `{..}` is different than anywhere else.

Reply:
The issue is #3001, unsure why I cannot link it to the PR. It has:

> maybe even a stack trace into `TenantState::Broken`

I guess that's fine. Main point of access will probably be the http api.

Reply:
Yep, I'm ok with either. I can change the message to include only reason without a backtrace, will it be better?

Reply:
Confirm that it will return the backtrace in http handler